dashboard/app/views/pd/workshop_enrollment/show.html.haml

```diff
@@ -1,8 +1,12 @@
 %h1
   Thank you for registering.
 
-%p
-  You will receive a confirmation email.
+- if params.has_key?(:details)
+  = render partial: 'workshop_details'
+- else
+  %p
+    You will receive a confirmation email.
+
```

Comment:
This seems like odd logic for this view. What about a different route? Your usage actually seems more appropriate for `show`. Maybe the "Thank you for registering" and "You will receive a confirmation email" parts should only be present here with a param set that happens immediately after registration, or these should be separated into two views. Thoughts?